lib/rspec/page-regression/matcher.rb

```diff
@@ -9,10 +9,15 @@ module RSpec::PageRegression
 
       @responsive_filepaths = FilePaths.responsive_file_paths(RSpec.current_example, args)
 
-      opt = args.select { |k,_| RENDER_ARGS.include?(k) }
-
-      Renderer.render_responsive(page, @responsive_filepaths, opt)
-      @comparisons = @responsive_filepaths.map{ |filepaths| ImageComparison.new(filepaths) }
+      render_options = args.select { |k,_| RENDER_ARGS.include?(k) }
+      compare_options = {}
+      compare_options[:threshold] = args.fetch(:threshold, RSpec::PageRegression.threshold)
+      compare_options[:mode] = args.fetch(:mode, RSpec::PageRegression.mode)
+      compare_options[:include_rect] = args.fetch(:include_area, nil)
+      compare_options[:exclude_rect] = args.fetch(:exclude_rect, nil)
```

Comment:
The README above says "you can use all of matcher's options", but these lines hardwire support for exactly these 4 options.  I'd suggest adding another argument `imatcher: {options: 'would', go: 'here'}`, but with these four options supported as shortcuts.  that'd be easy to implement, just replace

```    compare_options={}```

with

```    compare_options = args.fetch(:imatcher, {})```

(And of course document that in the README.)  What do you think?